edx-olive/edx-platform
# [CAM2-353] [SE-2885] Update translations

The PR updates the Hebrew translation of these messages from the spreadsheet :
  - line 82, Please note … 
  - line 89, submit
  - 90 select an option
  - 91 bookmark this page
  - 95 Show Answer
  - 96 Add a Post
  - 97 Posts I'm Following
  - 98 Contact Us
  - 99 View Certificate
  - 100 Your certificate is available
  - 101 View Archived Course
  - 102 You can keep working for a higher grade, or request your certificate now

Also lines 92, 93, 94, which are harder:
- 93 and 94 are identical ("file") but it looks like they originally mention "SubRip" and "Text" respectively. To clarify whether both should be file. Not included in the PR yet
- Line 92 is for the left part ("Download"); this part is changed. The message is constructed [here](https://github.com/edx-olive/edx-platform/blob/bb9e5922e12c805668dc09697ccfa8623ec1d049/lms/templates/video.html#L64). It only required a small change. [It's this one](https://www.transifex.com/open-edx/open-edx-releases/translate/#he/release-ginkgo/116174471?q=text%3ADownload) in Transifex.

All messages were imported from Transifex and hand-picked (since Transifex imported many more changes than the ones in the spreadsheet).

# Screenshots of each message after applying the changes
And where to find them.

- line 82: support. Can be seen at https://courses.stage.campus.gov.il/courses/course-v1:edX+DemoX+Demo_Course/courseware/d8a6192ade314473a78242dfeedfbf5b/edx_introduction/, clicking the left bar. Attached screnshot:
![line82-supportlanguage](https://user-images.githubusercontent.com/132703/87253716-15e1c200-c486-11ea-9607-0461710a184e.png)

- line 89: submit. At https://courses.stage.campus.gov.il/courses/course-v1:edX+DemoX+Demo_Course/courseware/interactive_demonstrations/19a30717eff543078a5d94ae9d6c18a5/?child=first, multiple answers. Attached screnshot:
![line89-submit-button](https://user-images.githubusercontent.com/132703/87253717-1e39fd00-c486-11ea-8fa4-fa47250e46c7.png)

- line 90, dropdown („select an option“): I added a dropdown exercise to https://courses.stage.campus.gov.il/courses/course-v1:edX+DemoX+Demo_Course/courseware/interactive_demonstrations/19a30717eff543078a5d94ae9d6c18a5/ . Attached screnshot:
![line90-selectanoption-dropdown](https://user-images.githubusercontent.com/132703/87253719-22661a80-c486-11ea-9fce-49cafa97fe2b.png)

- line 91, bookmark this page, can be seen at https://courses.stage.campus.gov.il/courses/course-v1:edX+DemoX+Demo_Course/courseware/interactive_demonstrations/19a30717eff543078a5d94ae9d6c18a5/ at the top. Attached screenshot:
![line91-bookmarkthispage](https://user-images.githubusercontent.com/132703/87253722-25f9a180-c486-11ea-862b-f17f1bbf856e.png)

- line 92, download: It can be seen at https://courses.stage.campus.gov.il/courses/course-v1:opencraft+ACD_TAU_cs101x+2018_1/courseware/f86de2fcf6a441768a99fa66c05f425b/0359d5d5efe544b9bedc00770419a7d4/, the „download“ word is now „הורדת“ (earlier, it was „הורד“)
![line92-download](https://user-images.githubusercontent.com/132703/87253725-298d2880-c486-11ea-9d87-ef42e40af8c8.png)

- line 93, Text (.txt): to ask Campus whether this should be translated as just „קובץ“ (file) or rather „קובץ (.txt)“ or similar. Otherwise, both 93 and 94 are translated to the same string and they area near each other. Current status: see screenshot from 92

- line 94, SubRip (.srt): see line 93

- line 95, show answer (button), seen in many problem types, tested at https://courses.stage.campus.gov.il/courses/course-v1:edX+DemoX+Demo_Course/courseware/interactive_demonstrations/basic_questions/?child=first . Attached screenshot:
![line95-showanswer-button](https://user-images.githubusercontent.com/132703/87253734-3447bd80-c486-11ea-8725-0df9672fa47e.png)

- line 96, add a post. I saw it at https://courses.stage.campus.gov.il/courses/course-v1:OpenCraft+CS101+2020_T1d/discussion/forum/ . Attached screenshot. Note that the search button overlaps with the „support“ vertical bar at the left; scrolling the page solves it
![line96-addapost-button](https://user-images.githubusercontent.com/132703/87253737-37db4480-c486-11ea-8073-b340368c1ab6.png)

- line 97 posts I'm followng, seen at https://courses.stage.campus.gov.il/courses/course-v1:opencraft+ACD_TAU_cs101x+2018_1/discussion/forum/ . Attached screenshot. Note that another option is seen untranslated („General“) but it's not part of the messages pack we're updating
![line97-postsimfollowing](https://user-images.githubusercontent.com/132703/87253738-3b6ecb80-c486-11ea-8e7d-a7f6ae154241.png)

- line 98 contact us, at the bottom, for instance at https://courses.stage.campus.gov.il/courses/course-v1:opencraft+ACD_TAU_cs101x+2018_1/discussion/forum/ . Attached screenshot
![line98-contactus](https://user-images.githubusercontent.com/132703/87253740-3f9ae900-c486-11ea-9e42-cbb394047752.png)

- line 99 view certificate. I enabled certificates and finished a course and then in https://courses.stage.campus.gov.il/courses/course-v1:OpenCraft+CS101+2020_T1d/progress I clicked „Request certificate“. Then I saw the „View certificate“ button (99), „Your certificate is available“ (101) and „You can keep working for a higher grade […]“ (line 102). All these 3 messages. Attached screenshot:
![line99and100and102-gotcertificate](https://user-images.githubusercontent.com/132703/87253744-43c70680-c486-11ea-9879-daf7bd61c590.png)

- line 100 Your certificate is available: see line 99, included there

- line 101 view archived course, seen at https://courses.stage.campus.gov.il/dashboard . Attached screenshot
![line101-viewarchivedcourse-button](https://user-images.githubusercontent.com/132703/87253746-475a8d80-c486-11ea-9bf9-16a0a24d0562.png)

- line 102 „You can keep working for a higher grade, or request your certificate now“: see line 99, included there

# Test instructions
- review the screenshots above
- compare the new messages with the ones in the spreadsheet
- check that the PR includes all requested changes
- (I already did this): uncompile the .mo and also the previous .mo (with msgunfmt), and verify that the changes are just the strings mentioned in the .po and no more
- (I already did this to do the screenshots): check the locations of the text messages in Campus stage

# Deployment
This is already active in stage. Campus can review it there. After they do (and after we fix any detail like lines 93 and 94) we can merge and build an AMI and deploy to production.
CircleCI tests were failing for unrelated reasons ([e.g. timeouts](https://circleci.com/gh/edx-olive/edx-platform/475)); I re-run them. This doesn't prevent reviewing the changes in stage.
